wordpress-mobile/WordPress-Android
# Feature/android shortcuts

Fixes #4681

Adds Android Shortcuts for Stats, Notifcations and New Post screens.

To test:
1. Long press the launcher icon
2. Try to open the app using each of the shortcuts
3. Check whether the app is open on the correct screen

![shortcuts_1](https://user-images.githubusercontent.com/2261188/39171839-4532f4d0-47a0-11e8-8b01-48b5f49e1e06.png)
![shortcuts_2](https://user-images.githubusercontent.com/2261188/39171841-460fa268-47a0-11e8-9cc3-c583e3199796.png)

Note:
@iamthomasbishop Are names of these three events ok?
`"shortcut_stats_clicked", "shortcut_notifications_clicked", "shortcut_new_post_clicked"`
